lms/templates/ccx/grading_policy.html

```diff
@@ -3,8 +3,8 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
-      <h3 class="title">${_("WARNING")}</h3>
+    <div class="msg-content" tabindex="0">
```

Comment:
@aamir-khan, is this information appended to the DOM, or is it always there when you click on the Grading Policy tab?  If its always there, then the changes you need are not necessary. I screen reader user will encounter the error just like they would encounter all other content on the page.  It's important that the word "Warning" is in a heading element, which it already was.  There is no need to add a role=heading.  

If this content is appended to the DOM (injected) after page load, like if the user attempts to change something, what you want to do is add ```aria-live="polite"``` to the ```<div>``` into which the content is added.  The only important thing to consider here is that the ```<div>``` you are adding to is already on the page at page load.  It doesn't work if its on the ```<div>``` that actually gets appended to the DOM. 

It's also important to note that you should never include any interactive content, like a link, to an aria-live region since it doesn't have focus, its only read out loud.  It should also be relatively short.  This notice is short enough.  You can find out more about live regions here:

http://www.w3.org/WAI/PF/aria-practices/#LiveRegions

cc @pdpinch @pwilkins